website/docs/docs/dbt-cloud/on-premises/setup.md

```diff
@@ -0,0 +1,85 @@
+
+After configuring the application, you will be taken to the **Version History** page where you can manage which version of dbt Cloud is deployed. A series of preflight checks will run on your newly configured version. If everything is configured correctly, it will say "Ready to Deploy." Click Deploy to start the application.
+
+<img src="/img/docs/dbt-cloud/on-premises/version-history.png" />
+
+You can skip to the Deployment section below to learn more about how dbt Cloud deployment management works.
+
+### Create a dbt Cloud Account
+
+Within your dbt Cloud installation, you can create multiple Accounts that can have different user groups assigned to them. To create your first account, navigate to the dbt Cloud backend at `https://<hostname>/backend`. This is an administrative site you can use to manage the dbt Cloud application. You can login with username "superuser" and the Django Superuser Password you entered in the configuration console.
+
+After logging in, you can create a new account and invite members of your team, by doing the following:
+
+- Under Database > Accounts, click + Add
+- Enter a name for the account
+- Under Concurrency, click Show. Enter how many concurrent jobs to allow for this account. If you aren't sure, pick 1 for now.
+- Add users to the account (this is important -- if you don't add users, nobody will be able to access the account!)
+- If using Enterprise SSO, follow the integration-specific guide for setting up this account.
+- If using username / email login, you can invite users to this account by email from this page. Under Invites, enter the email address(es) of users you'd like to invite. When you save the account, these users will receive an invitation in their inbox.
+- Save the new account.
+
+### Account / User Concepts
+
+Accounts and Users are separate constructs that can have a many-to-many relationship. When creating a new Account, you can add either existing or new users to the account. If one user has access to multiple accounts, they will be able to switch accounts from the dbt Cloud frontend upon login.
+
+Each user can have a specific role on each account. For more information on each role, please see: https://docs.getdbt.com/docs/cloud-managing-permissions
+
+### Deployments
+
+A new version of dbt Cloud will appear on the Version History page in your Configuration Console anytime any of the following happen:
+
+- A new version of the dbt Cloud code is released. This typically happens once per week, and each new version will be accompanied by a changelog.
+- Any configuration change is applied to your application via the Configuration Console.
+- Anytime an edit is applied to your Kubernetes configs via the overlays mechanism built into kots.
+
+You can apply or roll back these changes at any time by clicking the Deploy and Rollback buttons on the right side of this screen.
+
+### Github
```

Comment:
You might want to note that this is only applicable to GitHub's self-hosted offering!

Reply:
@drewbanin that's not true! for onprem, customers can create their own Github app even in Github cloud-hosted